01_helloWorld/solution/helloWorld-solution.spec.js

```diff
@@ -1,7 +1,7 @@
-const helloWorld = require("./helloWorld");
+const helloWorld = require('./helloWorld-solution');
```

Comment:
Are all of the double quote to single quote changes just in the interest of consistency?

Does TOP have a style guide or some other policy that prefers single quotes?

Reply:
This repository is set up to use eslint with the airbnb style guide, but it's probably been a while since anyone actually ran this code through a linter.

Reply:
Unless... @thatblindgeye did you actually run eslint over this? :D

Reply:
@xandora all I did was save the file 😆

Reply:
I had a quick search through the main repository and curriculum repositories, and we don't explicitly mention that airbnb is our preferred style guide, but it is implied as part of the eslint config across the repositories.

* Contributions to the curriculum should follow the style guide mentioned in the [contributing guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md#top-curriculum-repo). 
* The [guide for contributions to the main site](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md#top-website-repo) doesn't specifically say "use airbnb", but we do have tests that must be passed which cover linting, so you don't need to think about it:
```
rubocop
yarn eslint
rspec
yarn test
```

We have no overall recommended list of extensions, except those that are mentioned during the curriculum at the appropriate time they need to be. There's no point in installing Prettier or Eslint in VSCode before we get to those lessons, as they will start clobbering a learners code in way they might not expect to see.

Reply:
Oh, didn't realize you get to that later in the curriculum. (I'm still going through it, approaching the end of the Foundations Course.) Sounds good, I can appreciate not wanting to overwhelm beginners.